pyanaconda/packaging/__init__.py

```diff
@@ -230,7 +230,7 @@ def _sourceNeedsNetwork(self, sources):
         """
         network_protocols = ["http:", "ftp:", "nfs:", "nfsiso:"]
         for s in sources:
-            if any(s.startswith(p) for p in network_protocols):
+            if any(s.startswith(p) for p in network_protocols if s):
```

Comment:
OMG yeah, I'm blind I thought I'm testing the internal for part. Thank you for pointing this out. I'll fix that immediately.